font/src/darwin/mod.rs

```diff
@@ -278,6 +281,25 @@ pub struct Font {
 
 unsafe impl Send for Font {}
 
+
+/// Set subpixel anti-aliasing on MacOS.
+pub fn set_font_smoothing(enable: bool) {
+    unsafe {
+        // MacOS Mojave 10.14.x disables subpixel anti-aliasing,
+        // to re-enable it on an application level, we want to ensure
+        // we are running at least 10.14.0.
+        let min_macos_version = NSOperatingSystemVersion::new(10, 14, 0);
+        if NSProcessInfo::processInfo(nil).isOperatingSystemAtLeastVersion(min_macos_version) {
```

Comment:
But in the case `enable` is false there is two scenarios: 

1. `CGFontRenderingFontSmoothingDisabled` is already set to `true` (default) because it has never been set to `false`.
2. `CGFontRenderingFontSmoothingDisabled` is set to `false` and in this case we want to remove our value (`false`) so it returns to `true`.

Reply:
No, in case the version isn't correct, we never want to do anything. Why would we try to remove a value we've never set ourselves?

Reply:
No, in case the version isn't correct, we never want to do anything. Why would we try to remove a value we've never set ourselves?

Reply:
Ah, alrite, I misunderstood you. This has been fixed.